core/services/standardcapabilities/delegate.go

```diff
@@ -114,6 +116,15 @@ func (d *Delegate) ServicesForSpec(ctx context.Context, spec job.Job) ([]job.Ser
 		return []job.ServiceCtx{capability, handler}, nil
 	}
 
+	if spec.StandardCapabilitiesSpec.Command == commandOverrideForCustomComputeAction {
+		computeSrvc := compute.NewAction(log, d.registry)
+		if err != nil {
```

Comment:
`compute.NewAction(log, d.registry) ` isn't returning an error, what error is being handled here?

Reply:
Yikes, thanks for catching!